server/models/models.py

```diff
@@ -0,0 +1,42 @@
+from server import db, flask_bcrypt, app
+import datetime
+import jwt
+
+
+class User(db.Model):
```

Comment:
Let's name this file user.py; we'll probably have more models eventually and will want to separate them. Then it'd be from models.user import User above in auth_handler